File: OEM/Domain/OperatorPlans/OperatorPlanDto.cs
using System;
using System.Collections.Generic;

namespace OEMAPI.Domain.OperatorPlans
{
    public class OperatorPlanDto
    {
        public Guid Id { get; set; }
        public string VvnId { get; set; }
        public DateTime CreatedAt { get; set; }
        public string CreatedBy { get; set; }
        public string AlgorithmUsed { get; set; }

        public List<Guid> PlannedOperationIds { get; set; } = new();
    }
}

File: OEM/src/dto/OperatorPlanDTO.ts
export interface OperatorPlanDTO {
  id: string;
  vveId: string;
  createdAt: string;
  createdBy: string;
  algorithmUsed: string;
}

export interface CreateOperatorPlanDTO {
  vveId: string;
  createdAt: string;
  createdBy: string;
  algorithmUsed: string;
}

export interface UpdateOperatorPlanDTO {
  createdAt?: string;
  createdBy?: string;
  algorithmUsed?: string;
}